cmd/internal/find/types/testdata/triangle.json

```diff
@@ -1,15 +1,18 @@
 {
     "A": {
+        "label": "A",
```

Comment:
Is there an example where `label` isn't redundant?

Reply:
Yes, see https://github.com/ekinanp/wash/blob/kind-primary-round2/cmd/internal/find/types/testdata/real_world_aws.json